hknweb/candidate/templates/candidate/index.html

```diff
@@ -220,10 +220,9 @@ <h4>Bit-Byte</h4>
                             <span class="req not-done"></span>
                         {% endif %}
                         {{ num_bitbytes }} Bit-Byte Activities Completed
-
+                        <br>
                         <a href="{% url 'candidate:bitbyte' %}">
-                            (View Details)</a>
-                        <ul>
+                            (View/Request Bit-Byte Confirmations)</a>
```

Comment:
I think you're aware of this, but maybe stick the entire 'Checklist' section in a table and make the table css gray background? Not sure if that helps, but maybe it'll work!

Reply:
Oh I fixed it by removing the upper limit on list item height